snmp/metadata.csv

```diff
@@ -130,7 +130,7 @@ snmp.cefcFanTrayOperStatus,gauge,,,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525]
 snmp.cfInterconnectStatus,gauge,,,,"[NetApp] The current status of the interconnect (notPresent(1), down(2), partialFailure(3), up(4)).",0,snmp,,
 snmp.cfwConnectionStatCount,rate,,,,[Cisco ASA 5525] This is an integer that contains the value of the resource statistic.,0,snmp,,
 snmp.cfwHardwareStatusValue,gauge,,,,[Cisco ASA 5525] This object contains the current status of the resource.,0,snmp,,
-snmp.chasEntPhysical,gauge,,,,"[ALCATEL-ENT1-CHASSIS-MIB] This table contains one row per physical entity. It is an extension for the entity physical table (rfc 2737) that is instantiated for every physical entity object. The fields are not always significant for every object. Make 'sum by {X}' queries" to count elements with the tag X.",0,snmp,,
+snmp.chasEntPhysical,gauge,,,,"[ALCATEL-IND1-CHASSIS-MIB] This table contains one row per physical entity. It is an extension for the entity physical table (rfc 2737) that is instantiated for every physical entity object. The fields are not always significant for every object. Make 'sum by {X}' queries" to count elements with the tag X.",0,snmp,,
```

Comment:
```suggestion
snmp.chasEntPhysical,gauge,,,,"[ALCATEL-IND1-CHASSIS-MIB] [ALCATEL-ENT1-CHASSIS-MIB] This table contains one row per physical entity. It is an extension for the entity physical table (rfc 2737) that is instantiated for every physical entity object. The fields are not always significant for every object. Make 'sum by {X}' queries" to count elements with the tag X.",0,snmp,,
```


Interesting, so, it's the same name used in both ALCATEL-ENT1-CHASSIS-MIB and ALCATEL-IND1-CHASSIS-MIB.

Reply:
Yes, ALCATEL-ENT1-CHASSIS-MIB is basically a new version of ALCATEL-IND1-CHASSIS-MIB. 
Alcatel migrated from IND to ENT and changed sysobjectids at the same time to avoid breaking changes in Network Monitoring systems, which is very clever 🙂

[Source : release note 218556](https://support.alcadis.nl/Support_files/Alcatel-Lucent/OmniSwitch//OS6860(E)(N)/Firmware/OS6860(E)%20AOS%208.2.1.R01/OS6860(E)%20AOS%208.2.1.353.R01%20Maintenance%20Release/OS6860(E)%20AOS%208.2.1.353.R01%20Maintenance%20Release%20Notes.pdf)